lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1045,7 +1037,15 @@ def _embedTeXFont(self, fontinfo):
             self.type1Descriptors[(fontinfo.fontfile, effects)] = fontdesc
         fontdict['FontDescriptor'] = fontdesc
 
+        # Use TeX Font Metrics file to get glyph widths (TeX uses its 12.20 fixed point
+        # representation and we want 1/1000 text space units)
+        tfm = fontinfo.dvifont._tfm
+        widths = [(1000 * metrics.tex_width) >> 20
+                  if (metrics := tfm.get_metrics(char)) else 0
+                  for char in range(fc, lc + 1)]
+        fontdict['Widths'] = (widthsObject := self.reserveObject('glyph widths'))
```

Comment:
This walrus operator appears superfluous.
```suggestion
        fontdict['Widths'] = widthsObject = self.reserveObject('glyph widths')
```